src/sig3.iced

```diff
@@ -47,6 +48,11 @@ exports.OuterLink = class OuterLink
       schema.link_type().name("link_type")
       schema.chain_type().name("chain_type")
       schema.bool().name("ignore_if_unsupported")
+      schema.dict({
+        k : schema.enc_kid().name("kid")
+        n : schema.binary(24).name("nonce")
```

Comment:
could this change if we migrate to another scheme? or should we just relax this when we do that

Reply:
yeah we'd have the version scheme there

Reply:
we'd need to change this little schema system to handle a variant or some such